msmtp.yaml

```diff
@@ -0,0 +1,60 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/tiff/APKBUILD
+package:
+    name: msmtp
+    version: 1.8.22
+    epoch: 0
+    description: msmtp is an SMTP client with a sendmail interface
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: |
+             Copyright (C) 2018, 2019, 2020, 2021, 2022
+             Martin Lambers <marlam@marlam.de>
+
+               This program is free software; you can redistribute it and/or modify
+               it under the terms of the GNU General Public License as published by
+               the Free Software Foundation; either version 3 of the License, or
+               (at your option) any later version.
+
+               This program is distributed in the hope that it will be useful,
+               but WITHOUT ANY WARRANTY; without even the implied warranty of
+               MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+               GNU General Public License for more details.
+
+               You should have received a copy of the GNU General Public License
+               along with this program.  If not, see <http://www.gnu.org/licenses/>.
+          license: GPL-3.0+
```

Comment:
shouldn't this be `GPL-3.0-or-later`? or did SPDX finally add the `GPL-3.0+` shorthand that some distros were pushing for?

Reply:
you're right. The most idiot thing is that I looked up the tag and copied it wrong. I need to go to bed.

Fixed

Reply:
Ah I see, I googled a deprecated page:  https://spdx.org/licenses/archive/archived_ll_v2.4/GPL-3.0+.html